src/spryker/application/overlay/config/install/docker.yml

```diff
@@ -0,0 +1,255 @@
+# This file is a copy of https://github.com/spryker-shop/b2c-demo-shop/blob/master/config/install/development.yml
+# We have remove US store from database related steps as original file expects a separate database for US store
+# which is not the case in this setup
+
+env:
+    APPLICATION_ENV: development
```

Comment:
Presuming this will need to change in the dynamic environments when we get there - a note for later.

Reply:
I will move it to a variable for now, so it can be changed easily later on